### PR TITLE
[MIG][14.0] base_search_custom_field_filter: Migration to 14.0

### PR DESCRIPTION
**Pull request by PierrickBrun**

### CODE REVIEW

File: base_search_custom_field_filter/README.rst [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--ux-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-ux/tree/13.0/base_search_custom_field_filter
```
Comment (remytms) [inline base_search_custom_field_filter/README.rst:17]: You should change `13.0` to `14.0`. Maybe try to re-generate readme ? :)

**Review by remytms — COMMENTED** [review]
I think that readme should be generated again. To get 14.0 links. Or perhaps it's the OCA bot that care about that ?

File: base_search_custom_field_filter/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply base_search_custom_field_filter/README.rst:17]: Bot does it on merge, although you can do it in advance for avoiding this kind of confusion.

**Review by hparfr — APPROVED** [review]

**Review by remytms — APPROVED** [review]
LGTM. :)

**Pull request merged**